docker-compose.yaml

```diff
@@ -0,0 +1,51 @@
+version: '2.3'
+
+services:
+  pd0:
+    image: pingcap/pd:latest
+    ports:
+      - "2379:2379"
+    volumes:
+      - ./config/pd.toml:/pd.toml:ro
+      - ./data:/data
+    command:
+      - --name=pd0
+      - --client-urls=http://0.0.0.0:2379
+      - --peer-urls=http://0.0.0.0:2380
+      - --advertise-client-urls=http://pd0:2379
+      - --advertise-peer-urls=http://pd0:2380
+      - --initial-cluster=pd0=http://pd0:2380
+      - --data-dir=/data/pd0
+      - --config=/pd.toml
+    restart: on-failure
+  tikv0:
```

Comment:
How do you solve startup sequence problem?
According to the compose file, it seems tikv will not wait for pd "ready to serve" and might fail to start.
That's what I encounter before using docker-compose with TiDB.

Reply:
There is a section `depends_on` which solve the issue you just mentioned.

Reply:
So now we use restart on failure to do "wait" logic, is that right?

Reply:
I think `depends_on` on line 34 is what does "wait" logic here.